defrag/modules/bugs.py

```diff
@@ -175,14 +172,12 @@ async def root() -> QueryResponse:
 
 
 @app.get("/" + __MOD_NAME__ + "/search/")
-async def search(query: BugzillaQueryEntry) -> QueryResponse:
```

Comment:
the deletion of these lines and the replacement with the lines until `login()` is not a satisfying solution I think. If it didn't cache, that's something I should fix, because ideally we would cache some bugs (I imagine that they are times were many people run into the same bugs, so that's a reason for caching now before having to refactor). So let me fix it tomorrow/today.

I agree however that the solution with partials might be avoided by changing slightly the design. Better now than too late.

As for login(), it should not exist where it is there. It should be tested for truth in every call. I'll try to include that in the small redesign.

Reply:
#49 
There are still these ugly comments 59-79. Do you have plans for them?

Reply:
Why do we even need to log in? Couldn't we just call `handler = bugzilla.Bugzilla(url=URL)`?

I'll remove the comments in 59-79 until I find a better solution for it.